docs/manuals/measurement/readout_mitigation.rst

```diff
@@ -102,9 +93,9 @@ The individual mitigation matrices can be read off the mitigator.
 
 .. jupyter-execute::
 
-    for m in mitigator._mitigation_mats:
```

Comment:
Besides changing the import location of the mitigators, I made some small changes to this manual to stop using private functions and methods.